src/js/components/midi_range_selector.js

```diff
@@ -0,0 +1,30 @@
+import React, { PropTypes } from 'react';
```

Comment:
@dbalatero 

:information_desk_person: babel generates something that looks like this whenever it sees jsx (i.e. XML acting like it belongs in JS) code:

```
(0, _reactDom.render)(React.createElement(
  _reactRedux.Provider,
  { store: store },
  React.createElement(_patch_app2.default, null)
```

pretty neat. this means that we have to `import React from 'react'` like this in any file dealing with with code that will be treated as a React component (even if it doesn't inherit explicitly from Component and is just a markup returning function, like on like 4 down there).

i think this is silly and we should consider using https://github.com/re5et/jsxon